scripts/importmetadata

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+# Generates an import format metadata file
+# Requires a built provider to reference, defaults to
+# $GOPATH/src/github.com/hashicorp/<tpg ga or beta>/...
+# 
+# Produces via following priorities:
+# Checks for all output .go resources with a declared Importer: in schema
+# Checks if it uses standard tpgresource.ParseImportId() functions, if so converts that regex
+# Checks if uses schema.ImportStatePassThrough(), if it has a hardcoded return based on how ID is constructed in the resource
+# Anything else is non standard -- will take from MD files 
+#
+# Expecting to see a resource and it's not being generated? Please reach out to @nickelliot
+
+shopt -s nullglob
+
+if [ -z "$1" ]
+then
+	echo "Please run script with the format './scripts/importmetadata <ga or beta> <basepath (optional arg)>'"
+	exit 1
+fi
+if [ -z "$2" ]
+then
+	basepath="$GOPATH/src/github.com/hashicorp/"
+else
+	basepath=$2
+fi
+if [ "$1"=="beta" ]
+then
+	basepath="${basepath}/terraform-provider-google-beta"
+	servicepath="${basepath}/google-beta"
+else
+	basepath="${basepath}/terraform-provider-google"
+	servicepath="${basepath}/google"
+fi
+
+echo "# $resource_type: [list-of-acceptable-import-templates]" > importmetadata.txt
```

Comment:
Should we write to stdout, and have the user pipe the input to a file of their choice? That'll make it more clear what we're doing in CI, rather than having a hidden file name chosen by the script.